tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# tasks file for ansible-playbook-telegraf
+#
+
+- name: fetch telegraf installation package
+  get_url: url=http://get.influxdb.org/telegraf/telegraf_{{ telegraf_version }}_amd64.deb dest=/tmp/telegraf_{{ telegraf_version }}_amd64.deb mode=0440
+
+- name: install telegraf package
+  command: dpkg --skip-same-version -i /tmp/telegraf_{{ telegraf_version }}_amd64.deb
+  register: dpkg_result
+  changed_when: "dpkg_result.stdout.startswith('Selecting')"
+
+- name: start telegraf service and ensure that it is enabled
+  service: name=telegraf enabled=yes state=started
```

Comment:
Would it be better if this came after the config task so that we don't start it twice (once with the default config)?